pkg/motor/create_account.go

```diff
@@ -46,6 +46,9 @@ func (mtr *motorNodeImpl) CreateAccount(request mt.CreateAccountRequest) (mt.Cre
 
 // CreateAccountWithKeys allows PSK and DSC to be provided manually
 func (mtr *motorNodeImpl) CreateAccountWithKeys(request mt.CreateAccountWithKeysRequest) (mt.CreateAccountWithKeysResponse, error) {
+	// Create Client instance
+	mtr.Cosmos = client.NewClient(client.ConnEndpointType_BETA)
```

Comment:
?

Reply:
This has to be done before `initMotor` now so that the client is ready to potentially query the address.